src/main/java/dev/jbang/source/KotlinScriptSource.java

```diff
@@ -0,0 +1,47 @@
+package dev.jbang.source;
+
+import static dev.jbang.net.KotlinManager.resolveInKotlinHome;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.function.Predicate;
+
+import org.jboss.jandex.ClassInfo;
+
+import dev.jbang.cli.BaseBuildCommand;
+import dev.jbang.net.KotlinManager;
+
+public class KotlinScriptSource extends ScriptSource {
+
+	protected KotlinScriptSource(ResourceRef script) {
+		super(script);
+	}
+
+	@Override
+	public List<String> getCompileOptions() {
+		return Collections.emptyList();
```

Comment:
should we add "KOTLINC_OPTIONS" here ? 

or maybe just "reuse" "JAVAC_OPTIONS" - is there ever a time where you need to specify/use both ?

Reply:
I can't imaging that most of the JAVAC_OPTIONS would even work with kotlinc. some would but many would not, i would think.  i agree with your latest commit to just leave them out completely until we have a sense of how someone wants/needs them.